<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(222, 222, 222);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Miscellaneous Parameters</string>
       </attribute>
       <widget class="QLineEdit" name="InputHeight">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>170</y>
          <width>154</width>
          <height>26</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>78</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Input Width</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="inputsize">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>130</y>
          <width>154</width>
          <height>26</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>83</width>
          <height>20</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Input Height</string>
        </property>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>486</y>
          <width>162</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>10</y>
          <width>251</width>
          <height>401</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
</string>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="submit">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>500</y>
          <width>133</width>
          <height>29</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Submit Parameters</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>420</y>
          <width>421</width>
          <height>112</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QGroupBox" name="inputtype">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(222, 222, 222);</string>
           </property>
           <property name="title">
            <string>Input Type</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QRadioButton" name="RGB">
              <property name="text">
               <string>RGB</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="GrayScale">
              <property name="text">
               <string>Gray Scale</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QGroupBox" name="arithmetictype">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(222, 222, 222);</string>
           </property>
           <property name="title">
            <string>Arithmetic Type</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QRadioButton" name="Float">
              <property name="text">
               <string>Float</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="Fixed">
              <property name="text">
               <string>Fixed</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(222, 222, 222);</string>
           </property>
           <property name="title">
            <string>Convolution Type</string>
           </property>
           <widget class="QRadioButton" name="Standard">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>50</y>
              <width>91</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text">
             <string>Standard </string>
            </property>
           </widget>
           <widget class="QRadioButton" name="Winograd">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>93</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text">
             <string>Winograd</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Data CSV Path</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>370</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Number of Epochs</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>330</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Learning Rate</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>290</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Batch Size</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>450</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Loss Function</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>410</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Optimizer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="datacsv">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>250</y>
          <width>154</width>
          <height>26</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="numEpochs">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>370</y>
          <width>154</width>
          <height>26</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="learningRate">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>330</y>
          <width>154</width>
          <height>26</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="batchSize">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>290</y>
          <width>154</width>
          <height>26</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_optimizer">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>410</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_lossFunc">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>450</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Architecture</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Layer</string>
              </property>
              <widget class="QComboBox" name="layer">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>40</y>
                 <width>171</width>
                 <height>26</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: white;</string>
               </property>
              </widget>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>190</y>
                 <width>750</width>
                 <height>331</height>
                </rect>
               </property>
               <property name="title">
                <string>Parameters</string>
               </property>
               <widget class="QWidget" name="layoutWidget">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>20</y>
                  <width>191</width>
                  <height>137</height>
                 </rect>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_4">
                   <item>
                    <widget class="QLabel" name="label_6">
                     <property name="text">
                      <string>Kernel Size</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_8">
                     <property name="text">
                      <string>No of Filters</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_9">
                     <property name="text">
                      <string>Stride</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_10">
                     <property name="text">
                      <string>No of Units</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_5">
                   <item>
                    <widget class="QLineEdit" name="kernelsize">
                     <property name="styleSheet">
                      <string notr="true">background-color: white;</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="filters">
                     <property name="styleSheet">
                      <string notr="true">background-color: white;</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="stride">
                     <property name="styleSheet">
                      <string notr="true">background-color: white;</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="units">
                     <property name="styleSheet">
                      <string notr="true">background-color: white;</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
               <widget class="QGroupBox" name="padding">
                <property name="geometry">
                 <rect>
                  <x>30</x>
                  <y>180</y>
                  <width>151</width>
                  <height>91</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(222, 222, 222);</string>
                </property>
                <property name="title">
                 <string>Padding</string>
                </property>
                <widget class="QWidget" name="layoutWidget">
                 <property name="geometry">
                  <rect>
                   <x>20</x>
                   <y>30</y>
                   <width>108</width>
                   <height>28</height>
                  </rect>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QRadioButton" name="yes">
                    <property name="text">
                     <string>Yes</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="no">
                    <property name="text">
                     <string>No</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
               <widget class="QWidget" name="layoutWidget">
                <property name="geometry">
                 <rect>
                  <x>240</x>
                  <y>260</y>
                  <width>244</width>
                  <height>31</height>
                 </rect>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QPushButton" name="layerbutton">
                   <property name="text">
                    <string>Submit Layer</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="Arch">
                   <property name="text">
                    <string>Submit Architecture</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
              <widget class="QTableView" name="tableView">
               <property name="geometry">
                <rect>
                 <x>275</x>
                 <y>31</y>
                 <width>461</width>
                 <height>351</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: white;</string>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QAbstractScrollArea::AdjustToContents</enum>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Output Files</string>
       </attribute>
       <widget class="QListWidget" name="listWidget">
        <property name="geometry">
         <rect>
          <x>31</x>
          <y>52</y>
          <width>261</width>
          <height>391</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>20</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Generated Files</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>20</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Code Sample</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>460</y>
          <width>195</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="generate">
           <property name="text">
            <string>Generate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>51</y>
          <width>411</width>
          <height>391</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
